octopod-backend/src/Octopod/Server/ControlScriptUtils.hs

```diff
@@ -14,6 +14,7 @@ module Octopod.Server.ControlScriptUtils
   , runCommandArgs
   , runCommandArgs'
   , checkCommandArgs
+  , checkArchiveArgs
```

Comment:
This is args for script called `archive check` [docs](https://github.com/typeable/octopod/blob/master/docs/en/Control_scripts.md#-archive_check)